src/components/Round.vue

```diff
@@ -174,9 +172,13 @@ getRound();
     </div>
 
     <h3 class="text-white text-xl my-4">
-      Spectator View
+      Broadcast Round Embed
```

Comment:
I don't think the title is necessary. Also the embed should probably not be inside an `h3` element.